### PR TITLE
Nazar/feature/create recipes

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: app/models/recipe.py [hunk 1]
```diff
@@ -31,12 +33,15 @@ class Recipe(db.Model, ModelMixin):
     updated_at: orm.Mapped[datetime] = orm.mapped_column(default=datetime.utcnow, onupdate=datetime.utcnow)
     is_deleted: orm.Mapped[bool] = orm.mapped_column(default=False)
 
-    cooking_time: orm.Mapped[int] = orm.mapped_column(default=0)
+    cooking_time: orm.Mapped[float] = orm.mapped_column(default=0)
```
Comment (Taraslysak) [inline app/models/recipe.py]: Why float? To represent hours? I'd suggest to have this value as integer and represent seconds, so you could calculate minutes and hours easily

File: app/views/category.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    )
+
+
+@bp.route("/create", methods=["GET", "POST"])
+@login_required
+def create():
+    form = f.CategoryForm()
+
+    if form.validate_on_submit() and not db.session.scalar(
+        sa.select(m.Category.name).where(m.Category.name == form.name.data)
```
Comment (Taraslysak) [inline app/views/category.py:43]: And what if we try to re-create deleted category?

File: app/views/recipe.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        flash("Recipe added!", "success")
+        recipe.save()
+        log(log.INFO, "Form submitted. Recipe: [%s]", recipe)
+        return redirect(url_for("recipe.get_all"))
+    if form.errors:
+        log(log.INFO, "Form error [%s]", form.errors)
+        flash(f"{form.errors}", "danger")
+        return redirect(url_for("recipe.get_all"))
+
+    return render_template("recipe/form.html", form=form)
```
Comment (Taraslysak) [inline app/views/recipe.py:80]: So if we will try to create a recipe with the name which is already in our DB, user will have no indication about it?

File: app/views/recipe.py [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+        form.description.data = recipe.description
+        form.plant_varieties.data = [pv.name for pv in recipe.plant_varieties]
+        form.categories.data = [c.name for c in recipe.categories]
+
+    if (
+        request.method == "POST"
+        and form.validate_on_submit()
+        and not db.session.scalar(
+            sa.Select(m.Recipe.name).where(m.Recipe.name == form.name.data, m.Recipe.uuid != uuid)
+        )
```
Comment (Taraslysak) [inline app/views/recipe.py]: Feels like user won't get any feedback if he will end up in this scenario

**Review by Taraslysak — COMMENTED** [review]
Good job, please consider some refactoring

File: app/views/category.py (hunk 2 above)
Comment (Nazar-kob) [reply app/views/category.py:43]: we don't have the option to delete the category so we can keep it as is

File: app/views/recipe.py (hunk 3 above)
Comment (Nazar-kob) [reply app/views/recipe.py:80]: solved

**Pull request merged**